plugins/filter_aws/aws.c

```diff
@@ -102,6 +104,18 @@ static void expose_aws_meta(struct flb_filter_aws *ctx)
                     "aws." FLB_FILTER_AWS_HOSTNAME_KEY,
                     ctx->hostname);
     }
+
+    for (i = 0; i < ctx->tags_count; i++) {
+        /* len("aws.ec2.tags.") = 13 */
+        env_tag_name = flb_sds_create_size(13 + ctx->tag_keys_len[i]);
+        if (!env_tag_name) {
+            flb_plg_error(ctx->ins, "failed to allocate memory for env tag name");
+            continue;
+        }
+        env_tag_name = flb_sds_printf(&env_tag_name, "aws.ec2.tags.%s", ctx->tag_keys[i]);
```

Comment:
`flb_sds_printf ` can also alloc so you need a check after it and an flb_errno() call if it failed.

Reply:
Done, https://github.com/fluent/fluent-bit/pull/6065/commits/10bf11978250daa614a75fae24b6e781910ec1cd